admin/views/service-manager/check-windows-form.ejs

```diff
@@ -48,59 +48,59 @@
                 </label>
                 <div class="form-date">
                     <div class="form-group form-group-day">
                         <label class="form-label" for="adminStartDay">Day</label>
-                        <input class="form-control <%= error.adminStartDay ? 'form-control-error' : '' %>"
+                        <input class="form-control <%= error.isError('adminStartDay') ? 'form-control-error' : ''%>"
                                id="adminStartDay"
                                name="adminStartDay"
                                type="number"
-                               value="<%= checkWindowData.adminStartDay %>"
+                               value="<%= typeof checkWindowData !== 'undefined' && checkWindowData.adminStartDay %>"
                                aria-required="true"
                                aria-label="Administration start day."
-                               <% if (adminIsDisabled > 0) { %>
+                               <% if (adminIsDisabled) { %>
                                disabled="disabled"
                                <% } %>>
                     </div>
                     <div class="form-group form-group-month">
                         <label class="form-label" for="adminStartMonth">Month</label>
-                        <input class="form-control <%= error.adminStartMonth ? 'form-control-error' : '' %>"
+                        <input class="form-control <%= error.isError('adminStartMonth') ? 'form-control-error' : '' %>"
                                id="adminStartMonth"
                                name="adminStartMonth"
                                type="number"
-                               value="<%= checkWindowData.adminStartMonth %>"
+                               value="<%= typeof checkWindowData !== 'undefined' && checkWindowData.adminStartMonth %>"
                                aria-required="true"
                                aria-label="Administration start month."
-                               <% if (adminIsDisabled > 0) { %>
+                               <% if (adminIsDisabled) { %>
                                disabled="disabled"
                                <% } %>>
                     </div>
                     <div class="form-group form-group-year">
                         <label class="form-label" for="adminStartYear">Year</label>
-                        <input class="form-control <%= error.adminStartYear ? 'form-control-error' : '' %>"
+                        <input class="form-control <%= error.isError('adminStartYear') ? 'form-control-error' : '' %>"
                                id="adminStartYear"
                                name="adminStartYear"
                                type="number"
-                               value="<%= checkWindowData.adminStartYear %>"
+                               value="<%= typeof checkWindowData !== 'undefined' && checkWindowData.adminStartYear %>"
```

Comment:
Why do we need all these checks ` typeof checkWindowData !== 'undefined'`?

Reply:
@jon-shipley updated